app/views/communities/_form.html.erb

```diff
@@ -39,7 +41,8 @@
         <div class="create-post m-3">
           <div class="form-group">
             <%= label_tag :category %>
-            <%= form.select :category, Community::CATEGORIES, class: "form-control" %>
+            <%= form.select :category, Community::CATEGORIES, prompt: "Select Category" ,class: "form-control "+is_error(@community,(:category)).to_s %>
```

Comment:
you can move the error part to a helper?

Reply:
same here also @iMagesh